.github/workflows/tests_linux.yml

```diff
@@ -630,13 +673,15 @@ jobs:
           cp -rf ${{ github.workspace}}/Kokkos_install/${{ matrix.exec_model }}/* Kokkos/
 
       - name: Install dependencies
-        run: sudo apt-get update && sudo apt-get -y -q install cmake gcc-$GCC_VERSION  g++-$GCC_VERSION  ninja-build gcovr lcov
+        run: sudo apt-get update && sudo apt-get -y -q install cmake gcc-$GCC_VERSION  g++-$GCC_VERSION liblapack-dev ninja-build gcovr lcov 
 
       - name: Build and run unit tests
         run: |
+            set -x
```

Comment:
why is this required?

Reply:
We enable the debug mode here to allow us to track where does the failure come from.